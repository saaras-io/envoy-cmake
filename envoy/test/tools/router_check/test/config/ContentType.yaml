virtual_hosts:
- name: local_service
  domains:
  - '*'
  routes:
  - match:
      prefix: "/"
      headers:
        - name: "content-type"
          exact_match: "application/grpc"
    route:
      cluster: local_service_grpc
  - match:
      prefix: "/"
    route:
      cluster: local_service
