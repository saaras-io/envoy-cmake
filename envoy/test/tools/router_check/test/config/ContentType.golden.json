[
  {
    "test_name": "Test_1",
    "input": {
      ":authority": "www.lyft.com",
      ":path": "/"
    },
    "validate": {"cluster_name":"local_service"}
  },
  {
    "test_name": "Test_2",
    "input": {
      ":authority": "www.lyft.com",
      ":path": "/",
      "additional_headers": [{"field": "content-type", "value": "application/grpc"}]
    },
    "validate": {"cluster_name": "local_service_grpc"}
  },
  {
    "test_name": "Test_3",
    "input": {
      ":authority": "www.lyft.com",
      ":path": "/",
      "additional_headers": [{"field": "content-type", "value": "foo"}]
    },
    "validate": {"cluster_name": "local_service"}
  }
]
